### PR TITLE
Upload code coverage from azure

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
Hmmm realized that we need to skip coverage for `pypy` because it is way too slow. It has been running now for 30m, and only reached 60% of the tests. 😓

**Comment by nicoddemus** [comment]
Ready for review.

Hmm bummer, build time jumped from 15m to 30m because of coverage. 😞

File: azure-pipelines.yml [hunk 1]
```diff
@@ -99,3 +103,8 @@ jobs:
       testResultsFiles: 'build/test-results/$(tox.env).xml'
       testRunTitle: '$(tox.env)'
     condition: succeededOrFailed()
+
+  - script: call scripts\upload-coverage.bat
+    displayName: 'Upload coverage'
+    env:
+      CODECOV_TOKEN: $(CODECOV_TOKEN)
```
[the remark this answers is not in the document]
Comment (nicoddemus) [reply azure-pipelines.yml]: Yep, as a [secret variable](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#secret-variables). 👍

**Comment by nicoddemus** [comment]
@blueyed I'm not sure the flags adjustment to codecov helped, the `codecov/project` check is still timing out for me. 😞

**Comment by blueyed** [comment]
@nicoddemus 
too bad, sorry.
It is likely more/also related to the number of uploads (which we now have increased with azure).

**Comment by blueyed** [comment]
The commit itself can be seen on https://codecov.io/gh/pytest-dev/pytest/tree/5494eb1948351d44ff58288dcfc500d1a584aed6 - but is not having the "Changes".

Those are due to skipping Linux/MacOS/Travis though likely, no?

**Comment by blueyed** [comment]
I'll go ahead and amend this.

**Comment by nicoddemus** [comment]
> Those are due to skipping Linux/MacOS/Travis though likely, no?

Right, most likely. 

> I'll go ahead and amend this.

OK, thanks!

**Comment by blueyed** [comment]
@nicoddemus 
I think you want to drop AppVeyor completely soon now, but we should remove coverage reporting from it at least as soon as possible.

**Comment by nicoddemus** [comment]
Yeah not sure how much `py37-freeze` is a blocker on Azure anyway. I will give it a shot later, and if I can't get it to work I think it is fine to just assume Azure officially and drop AppVeyor.

**Pull request merged**